salt/version.py

```diff
@@ -652,7 +651,9 @@ def __get_version(saltstack_version):
         from salt._version import __saltstack_version__  # pylint: disable=E0611,F0401
 
         return __saltstack_version__
+
     except ImportError:
+        ## except ImportError as exc:
```

Comment:
Is this comment required?

Reply:
left it as a reminder, given will will need to handle ImportErrors once we move to using pyproject, and packaging.version, or other imports with pip.  Took me a few days to eventually get to the source of the issue with 3006.0 being returned as the version, and it was here to to ImportError.

So not really required, but someone looking at hopefully will get triggered by having to maybe handle an exception here.  Can remove it if necessary,, or perhaps a better comment.